application/core/LSYii_FilterValidator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+if (!defined('BASEPATH')) {
+    exit('No direct script access allowed');
+}
+/*
+ * LimeSurvey
+ * Copyright (C) 2007-2011 The LimeSurvey Project Team / Carsten Schmitz
+ * All rights reserved.
+ * License: GNU/GPL License v2 or later, see LICENSE.php
+ * LimeSurvey is free software. This version may have been modified pursuant
+ * to the GNU General Public License, and as distributed it includes or
+ * is derivative of works licensed under the GNU General Public License or
+ * other free or open source software licenses.
+ * See COPYRIGHT.php for copyright notices and details.
+ *
+ */
+
+/**
+ * Extends CFilterValidator class to add PHP 8.1 compatibility. Based on Yii 2's FilterValidator
+ * @inheritdoc
+ */
+class LSYii_FilterValidator extends CFilterValidator
```

Comment:
Same question here, it's not in any Yii 1.1 patch? Did you submit a patch to Yii 1.1?

Reply:
If I am not wrong, I think we did this before Yii had any support for PHP8.1.
I could be wrong though.

Any way, We should check, agree